server/routes/shared_api/place.py

```diff
@@ -697,6 +690,16 @@ def placeid2dcid():
   return Response(json.dumps(result), 200, mimetype='application/json')
 
 
+@bp.route('/placeid2dcid')
```

Comment:
Is this endpoint still needed?

Reply:
The legacy search still uses it. I wouldnt want to break it as part of this PR but im not sure whether it still gets used? 
So I wasnt planning on removing the route just yet.

Reply:
Sounds good!